src/stdlib/crc32.rs

```diff
@@ -0,0 +1,77 @@
+use crate::compiler::prelude::*;
+use crc32fast::Hasher;
+
+fn crc32(value: Value) -> Resolved {
+    let value = value.try_bytes()?;
+    let mut hasher = Hasher::new();
+    hasher.update(&value);
+    Ok(format!("{:x}", hasher.finalize()).into())
+}
+
+#[derive(Clone, Copy, Debug)]
+pub struct Crc32;
+
+impl Function for Crc32 {
+    fn identifier(&self) -> &'static str {
+        "crc32"
+    }
+
+    fn parameters(&self) -> &'static [Parameter] {
+        &[Parameter {
+            keyword: "value",
+            kind: kind::ANY,
```

Comment:
Hm I noticed this in other hash function. I don't think `ANY` is ideal but we can leave it here for consistency.